### DISCUSSION TITLE
Contributions to the project

### DISCUSSION THREAD

**Discussion by ghost** [Blackymas/NSPanel_HA_Blueprint#222] [General]
Hi Team, great project! 

I'd love to help where I can, I can work on the EN docs if still needed and also have plenty of experience with ESPHome / HMI side if I can be of any use on the development? 

Thanks 
Dan

**Comment by Blackymas** [comment]
Hello sounds good.

You only speak EN? Should not be a problem but purely out of interest

**Comment by edwardtfn** [comment]
I'd like to help on my free time also... I have 2 panels, one EU type and another one a US type sitting on my workstation (which could easily be used for testing as no one else at home use it).
I speak English and have Brazilian Portuguese as native language.

Just let me know how to help. ;)
